extensions/wikia/Bucky/js/spec/bucky_resources_timing.spec.js

```diff
@@ -34,6 +34,22 @@ describe('BuckyResourcesTiming', function () {
 		});
 	});
 
+	it('correctly get the domain from URL', function() {
```

Comment:
how about test for a URL that wouldn't be valid so we make sure this will be handled properly and doesn't break anything else?